### ISSUE TITLE
Sorting by likes should sort by the different between likes and dislikes

### ISSUE THREAD

**Issue by antonkomarev** [cybercog/laravel-love#4]
Original issue from @sroy:

> There is no way to sort by ( likes - dislikes ) that I can find currently, only by total amount of likes and total amount of dislikes. Something to sort by likesDiffDislikesCount() would be really nice :) I'm not experienced enough send a PR.

To implement this feature we will require to introduce additional table or add extra rows in `like_counter` table with additional enum type related to `likes_diff_dislikes`. Otherwise queries and sorting in runtime will be significantly slow on big databases.

**Comment by a-komarev** [comment]
Another one naming could be `likes_vs_dislikes`.

**Comment by antonkomarev** [comment]
This feature will be available after PR #24 will be merged.

```php
$articlesQuery = Article::withReactionSummary();
// Scope will add virtual column `reactions_total_weight`
// `reactions_total_weight` equals to `likes weight` diff `dislikes weight`
// and then just simply order your models
$orderedArticles = $articlesQuery->orderBy('reactions_total_weight', 'desc');
```
